src/sentry/monitors/logic/mark_ok.py

```diff
@@ -28,9 +28,6 @@ def mark_ok(checkin: MonitorCheckIn, succeeded_at: datetime) -> None:
     """
     monitor_env = checkin.monitor_environment
 
-    if monitor_env is None:
-        return None
```

Comment:
This code will be active *before* the migration is run. Will that behave correctly?

Reply:
it's never None so it's currently dead code

Reply:
Yep, what Anthony said.